xml/docu_styleguide.structure.xml

```diff
@@ -1118,22 +1118,21 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
    </listitem>
    <listitem>
     <para>
-     In identifiers, only use alphanumeric characters, <literal>.</literal>,
-     <literal>_</literal>, and <literal>-</literal>.
+     In identifiers, only use alphanumeric characters and <literal>-</literal>.
     </para>
    </listitem>
    <listitem>
     <para>
-     Identifiers consist of up to three parts. Join these parts with a
-     <literal>.</literal>.
+     Identifiers can consist of multiple parts. Join these parts with a
+     <literal>-</literal>.
```

Comment:
I would add a note here, explicitly mentioning to *not* use dashes or underscores because of search engine optimization.

Reply:
Done (above)